bigbluebutton-html5/imports/ui/components/video-provider/component.jsx

```diff
@@ -110,7 +111,7 @@ class VideoProvider extends Component {
     };
 
     // Set a valid bbb-webrtc-sfu application server socket in the settings
-    this.ws = new ReconnectingWebSocket(Meteor.settings.public.kurento.wsUrl);
+    this.ws = new ReconnectingWebSocket(Auth.authenticateURL(Meteor.settings.public.kurento.wsUrl));
```

Comment:
If I revert this line I am still able to use webcam :man_shrugging: Doesn't this mean we didn't auth?
I have applied the nginx and tomcat changes.

Reply:
I was missing 2 lines in the sfu nginx config